nest-demo/otel-demo/load_test/load_test.sh

```
@@ -0,0 +1,2 @@
+# This script runs a load test using k6 on the specified JavaScript file.
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)